test/unit/core/error_helpers.js

```diff
@@ -1,4 +1,94 @@
 suite('Error Helpers', function() {
+  var myp5 = new p5(function( sketch ) {
+    sketch.setup = function() {};
+    sketch.draw = function() {};
+  });
+  var c;
+
+  // unit tests for validateParameters
+  suite('validateParameters', function(){
+    test('documentation look up - arc()', function() {
+      assert.doesNotThrow(function() {
+          p5.prototype._validateParameters('arc',
+            [1, 1, 10.5, 10, 0, Math.PI, 'pie']);
+        },
+        Error, 'got unwanted exception');
+    });
+    test('documentation look up - arc()', function() {
+      assert.doesNotThrow(function() {
+          p5.prototype._validateParameters('arc',
+            [1, 1, 10.5, 10]);
+        },
+        Error, 'got unwanted exception');
+    });
```

Comment:
This generates console.log err msg:
`> p5.js says: FES: It looks like arc received an empty variable in spot #4 (zero-based index). If not intentional, this is often a problem with scope: [link to scope]. [http://p5js.org/reference/#p5/arc]
p5.js:42683 > p5.js says: FES: It looks like arc received an empty variable in spot #5 (zero-based index). If not intentional, this is often a problem with scope: [link to scope]. [http://p5js.org/reference/#p5/arc]`